### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-it/content_conversationlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#226]
```
Added in italian translation conversationlis
conversationlist_bwm_agent_1
conversationlist_bwm_agent_2
conversationlist_bwm_agent_3
conversationlist_bwm_agent_4
conversationlist_bwm_agent_5
conversationlist_bwm_agent_6
conversationlist_prim_arghest
conversationlist_prim_outside
conversationlist_prim_inn
conversationlist_prim_tavern
conversationlist_prim_guthbered
conversationlist_blackwater_signs

This game is becoming really nice!
 great oskar, keep it up!!!

```

Original issue reported on code.google.com by `k6b...@gmail.com` on 30 Jul 2011 at 12:50

Attachments:
- [content_conversationlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-226/comment-0/content_conversationlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Issue 229 replaces this one, I presume?
```

Original comment by `oskar.wi...@gmail.com` on 30 Oct 2011 at 1:12
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.11**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Issue 229 was committed in r194. Marking this as ready.
```

Original comment by `oskar.wi...@gmail.com` on 30 Oct 2011 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
